packages/format-library/src/link/index.js

```diff
@@ -157,7 +184,7 @@ function Edit( {
 				aria-haspopup="true"
 				aria-expanded={ addingLink }
 			/>
-			{ addingLink && (
+			{ ( ( addingLink && isActive ) || addingNewLink ) && (
```

Comment:
addingLink || addingNewLink reads oddly. It's not clear what the difference is.

Reply:
Not yet 😅

Reply:
Now fixed.